### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Accessibility improvements

### Related
**[PT-510](https://helsinkisolutionoffice.atlassian.net/browse/PT-510):**

### CODE REVIEW

File: src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx [hunk 1]
```diff
@@ -162,6 +173,7 @@ const EnrolmentTable: React.FC<Props> = ({
               'occurrenceDetails.enrolmentTable.showEnrolmentDetails'
             )}
             {...row.getToggleRowExpandedProps()}
+            aria-expanded={row.isExpanded ? true : false}
```
Comment (jorilindell) [inline src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx:176]: isExpanded already sounds like a boolean value.

**Review by jorilindell — APPROVED** [review]
Looks good

File: src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx (hunk 1 above)
Comment (Sartonon) [reply src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx:176]: yeah, for some reason isExpanded is undefined when closed, so had to do this

Comment (jorilindell) [reply src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx:176]: Ok, makes sense

Comment (Sartonon) [reply src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx:176]: But maybe comment would be nice here

**Pull request merged**